src/test/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginTest.groovy

```diff
@@ -103,6 +104,38 @@ class ScalafixPluginTest extends Specification {
         }
     }
 
+    @Unroll
+    def 'The plugin uses semanticdb version #semanticdbVersion when provided using the scalafix extension'() {
+        given:
+        applyScalafixPlugin(scalaProject, true)
+        scalaProject.scalafix.semanticdbVersion = semanticdbVersion
+
+        when:
+        scalaProject.evaluate()
+
+        then:
+        scalaProject.tasks.scalafixMain // forces plugin configuration
+        def compileScalaParameters = scalaProject.tasks.compileScala.scalaCompileOptions.additionalParameters
+        compileScalaParameters.find {
+            it.startsWith('-Xplugin:') &&
+                    it.endsWith(expectedSemanticdbJar) &&
+                    !it.contains("scala-library")
+        }
+
+        scalaProject.tasks.scalafixTest // forces plugin configuration
+        def compileTestScalaParameters = scalaProject.tasks.compileTestScala.scalaCompileOptions.additionalParameters
+        compileTestScalaParameters.find {
+            it.startsWith('-Xplugin:') &&
+                    it.endsWith(expectedSemanticdbJar) &&
+                    !it.contains("scala-library")
+        }
+
+        where:
+        semanticdbVersion   || expectedSemanticdbJar
+        '4.4.9'             || "semanticdb-scalac_${SCALA_VERSION}-${semanticdbVersion}.jar"
+        '4.4.10'            || "semanticdb-scalac_${SCALA_VERSION}-${semanticdbVersion}.jar"
```

Comment:
I'd probably not use `@Unroll` here as we don't really care what the versions are. Maybe we could just pass some bogus version number (e.g. `9.9.9`) and have the assertions expecting that value only

Reply:
hehe, that was my first thought. I tried with 1.2.3, which didn't work as it is not a valid version.

I just used `@Unroll` there because I want to make sure the version is actually changing and that it is not picking up the default version provided by the plugin as there could be a chance the plugin is providing the same version you're setting in the extension.

Reply:
ok, I get you. The plugin tries to resolve the dependency when this test is run